boards/esp32-olimex-evb/Makefile.features

```diff
@@ -2,7 +2,6 @@
 include $(RIOTBOARD)/common/esp32/Makefile.features
 
 # additional features provided by the board (no ADC and no DAC)
-FEATURES_PROVIDED += periph_adc
```

Comment:
Why is this being removed?

Reply:
Since the board doesn't have GPIO left that might be used as ADC channels in default configuration. This is at least true for the Olimex ESP32-EVB Gateway board version.

Reply:
In `boards/esp32-olimex-evb/include/periph_conf.h` it seems that if `MODULE_OLIMEX_ESP32_GATEWAY` is defined some ADC pins are defined. Is that right?

Reply:
Yes, I can use `FEATURES_PROVIDED += periph_adc` dependent on whether module `olimex_esp32_gateway` is enabled.

Reply:
I have fixed it. If Olimex ESP32-EVB Gateway board is used, feature `adc` is provided.